### ISSUE TITLE
[Question]怎么在表格中添加一个自定义的button?

### ISSUE THREAD

**Issue by soluty** [GoAdminGroup/go-admin#187]
我希望在表格中自定义一个button进行操作，info.AddActionButton 只会在操作一栏显示，我希望是
info.AddField("head", "field", db.Varchar).FieldDisplay(func(value types.FieldModel) interface{} {
		return template.Default().
			Button().
			SetContent("我的按钮").
			GetContent()
	})
类似这种，但是ButtonAttribute不能添加action，所以这个按钮点击无效果，我要怎么做才能实现这个效果呢？

**Comment by chenhg5** [comment]
@soluty 当前版本没有API可以实现，明天会发新版本，可能会增加上这个功能API。

**Comment by devig** [comment]
Hi! Let me know, please. Does it ready now?

**Comment by shuiboshi** [comment]
有一个跟这个问题类似的问题，如下：
需求：两张表关联，如店铺表关联用户表（店主），在店铺表中希望点击店主跳转到用户表，或者查看店主信息
目前没有找到对应的实现的api，望添加此功能

**Comment by chenhg5** [comment]
@devig Yes, in the latest version of GoAdmin, use the [API](https://book.go-admin.cn/en/admin-plugins/data-table/basic#add-button): ```info.AddButton()```

**Comment by chenhg5** [comment]
@shuiboshi http://demo.go-admin.cn/admin/info/posts demo中可以找到类似你要的。
